lib/AST/Decl.cpp

```diff
@@ -7557,6 +7604,17 @@ bool FuncDecl::isMainTypeMainMethod() const {
          getParameters()->size() == 0;
 }
 
+bool VarDecl::isDistributedActorAddressName(ASTContext &ctx, DeclName name) {
+  assert(name.getArgumentNames().size() == 0);
+  return name.getBaseName() == ctx.Id_actorAddress;
+}
+
+bool VarDecl::isDistributedActorTransportName(ASTContext &ctx, DeclName name) {
+  assert(name.getArgumentNames().size() == 0);
+  return name.getBaseName() == ctx.Id_transport ||
+    name.getBaseName() == ctx.Id_actorTransport;
```

Comment:
// So this specific cunt I think I will remove entirely, we dont need it.

I think for the property we actually want `$transport` because it is synthesized; similar to how property wrappers do `$...`.

I think actors should do the same for their $executor too